toolchain/sem_ir/name_scope.h

```diff
@@ -18,13 +18,101 @@ enum class AccessKind : int8_t {
   Private,
 };
 
+// Represents the result of a name lookup.
```

Comment:
```suggestion
// Represents the result of a name lookup.
//
```

Blank line to separate paragraphs.

Reply:
Done.